appveyor.yml

```diff
@@ -1,22 +1,48 @@
-version: "{build}"
-build: off
-skip_tags: true
+build: false
+skip_branch_with_pr: true
 
 environment:
   matrix:
     - nodejs_version: "6"
     - nodejs_version: "8"
     - nodejs_version: "10"
+    - nodejs_version: "11"
+    - nodejs_version: "Current"
 
+configuration: Release
 platform:
   - x86
   - x64
 
 install:
+  # Workaround for https://github.com/appveyor/ci/issues/2420
+  - SET PATH=%PATH%;C:\Program Files\Git\mingw64\libexec\git-core
```

Comment:
I don't think we need this (the snappy submodule has been working fine in builds so far)